### PR TITLE
[Security] keys and keychains

### PR DESCRIPTION
**Pull request by sudhanshutech**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: content/en/cloud/security/keys.md [hunk 1]
```diff
@@ -1,238 +1,40 @@
 ---
 title: Keys
 description: >
-  A short lead description about this content page. It can be **bold** or _italic_ and can be split over multiple paragraphs.
+  Keys are a fundamental component of our cloud system, playing a crucial role in access control and permissions management. They are essential for ensuring that users have the appropriate access to specific features and functionalities within the system.
```
Comment (leecalcote) [inline content/en/cloud/security/keys.md:4]: This paragraph needs to be rewritten.

File: content/en/cloud/security/keys.md [hunk 2]
```diff
@@ -1,238 +1,40 @@
...
 
-{{% pageinfo %}}
-Page under construction.
-{{% /pageinfo %}}
 
-<!-- Text can be **bold**, _italic_, or ~~strikethrough~~. [Links](https://gohugo.io) should be blue with no underlines (unless hovered over).
+## Keys Management
 
-There should be whitespace between paragraphs. Vape migas chillwave sriracha poutine try-hard distillery. Tattooed shabby chic small batch, pabst art party heirloom letterpress air plant pop-up. Sustainable chia skateboard art party banjo cardigan normcore affogato vexillologist quinoa meggings man bun master cleanse shoreditch readymade. Yuccie prism four dollar toast tbh cardigan iPhone, tumblr listicle live-edge VHS. Pug lyft normcore hot chicken biodiesel, actually keffiyeh thundercats photo booth pour-over twee fam food truck microdosing banh mi. Vice activated charcoal raclette unicorn live-edge post-ironic. Heirloom vexillologist coloring book, beard deep v letterpress echo park humblebrag tilde.
+In our cloud system, a "key" is a unique identifier used to grant or restrict access to specific functionalities or features. Keys have a significant impact on access control, as they determine what users can or cannot do within the system.
```
Comment (leecalcote) [inline content/en/cloud/security/keys.md:14]: No: "in our cloud system".

File: content/en/cloud/security/keys.md [hunk 3]
```diff
@@ -1,238 +1,40 @@
...
 
-> There should be no margin above this first sentence.
->
-> Blockquotes should be a lighter gray with a border along the left side in the secondary color.
->
-> There should be no margin below this final sentence.
+### Key Name
 
-## First Header 2
+- It provides the clearest indication of the key's purpose.
```
Comment (leecalcote) [inline content/en/cloud/security/keys.md:20]: "It" what?

File: content/en/cloud/security/keys.md [hunk 4]
```diff
@@ -1,238 +1,40 @@
...
+   - Workspace Management > Environments:
+      Under the Workspace Management category, the Environments subcategory is used to manage environments.
 
-### Header 3
+### Description
 
-```
-This is a code block following a header.
-```
+- The description provides a more detailed explanation of the key's role.
```
Comment (leecalcote) [inline content/en/cloud/security/keys.md:39]: Include an example.

**Review by leecalcote — CHANGES_REQUESTED** [review]
Can keys be modified? Who can create them? How do I request new keys? 

What are they used for specifically? Are they grouped? Are they the same across all Meshery Servers? Across all Remote Providers? 

What types of keys are there? View, Edit, Delete, other? Where are keys enforced? When are keys updated? Does Cloud and Server have to be of the same version in order for keys to work? 

How do I review all keys? or view them by group? Can I create new groups of keys, if so how do I assign these to users? Can I assign keys directly? Can I assign keys to teams?

@theBeginner86, you can take over this PR.

**Comment by leecalcote** [comment]
@theBeginner86, you can take over this PR.

**Comment by leecalcote** [comment]
@theBeginner86 comfortable taking this on?

**Comment by theBeginner86** [comment]
> @theBeginner86, you can take over this PR.

Updated 👍

**Comment by theBeginner86** [comment]
I'm pressing forward here...

If there are issues or improvements, please let me know I'll come back and fix them. 👍

**Pull request merged**